README.md

```diff
@@ -6,6 +6,11 @@
 [![Release](https://github.com/bohning/usdb_syncer/actions/workflows/release.yaml/badge.svg)](https://github.com/bohning/usdb_syncer/actions/workflows/release.yaml)
 [![tox](https://github.com/bohning/usdb_syncer/actions/workflows/tox.yaml/badge.svg)](https://github.com/bohning/usdb_syncer/actions/workflows/tox.yaml)
 
+**USDB Syncer** is an app to download and synchronize UltraStar songs hosted on [USDB](https://usdb.animux.de).
+The project [extensively uses the `#VIDEO` tag](https://github.com/bohning/usdb_syncer/wiki/Meta-Tags#format) to automaticly retrieve the resources (audio, video, images, etc...) to make the UltraStar song complete.
+Once a song is downloaded it can be synchronized (new notes, audio, video, images...) by redownloading the song. If a resource didn't change it's skipped.
+The project can be used from the commandline, but also has a GUI available.
```

Comment:
What do you mean be commandline here? I'm not aware that we're supporting some kind of CLI.

Reply:
I think there's a cli-entrypoint right? (starting the gui from a commandline). So yeah i can imagine this line is a bit confusing. I'll remove it :)

Reply:
Yeah, that's just the entry point to the GUI. You can't do anything else with it.
Thanks.